code/obj/item/cameras.dm

```diff
@@ -85,6 +85,106 @@
 		desc = "No one cam should have all this power."
 		takes_voodoo_pics = 2
 
+/obj/item/camera/spy
+	inventory_counter_enabled = 1
+	var/obj/item/ammo/power_cell/self_charging/cell = null
+	var/flash_mode = 0
+	var/wait_cycle = 0
+
+	attack_self(mob/user)
+		if (user.mind.special_role == "spy_thief")
```

Comment:
don't like this, anyone should be able to do it, otherwise security confiscating stuff makes no sense

also having functionality locked to roles is dumb

Reply:
Check removed.